doc/source/reference/arrays.scalars.rst

```diff
@@ -274,7 +274,6 @@ The exceptions to the above rules are given below:
    generic
    generic.__array__
    generic.__array_wrap__
-   generic.squeeze
```

Comment:
I think removing this might cause the documentation for it to disappear. @mattip?

Reply:
Yes, I think this is needed.

Reply:
Thanks!  I reverted the change.